src/pages/Cart/Cart.css

```diff
@@ -0,0 +1,65 @@
+/* Cart.css */
+
+.cart-container {
+    padding: 20px;
+    background-color: #f9f9f9;
+    border: 1px solid #ddd;
+    border-radius: 5px;
+    margin-bottom: 20px;
+  }
+  
+  .cart-container h2 {
+    font-size: 1.5em;
+    margin-bottom: 20px;
+  }
+  
+  .cart-container p {
+    margin: 5px 0;
+  }
+  Cart.css */
+
+.cart-container {
+    padding: 20px;
+    background-color: #f9f9f9;
+    border: 1px solid #ddd;
+    border-radius: 5px;
+    margin-bottom: 20px;
+  }
+  
+  .cart-container h2 {
+    font-size: 1.5em;
+    margin-bottom: 20px;
+  }
+  
+  .cart-container p {
+    margin: 5px 0;
+  }
+  
+  .cart-container ul {
+    list-style-type: none;
+    padding: 0;
+  }
+  
+  .cart-container li {
+    border-bottom: 1px solid #ccc;
+    margin-bottom: 10px;
+    padding-bottom: 10px;
+  }
+  
+  .cart-container button {
+    margin-right: 10px;
+    background-color: #007bff;
+    color: white;
+    border: none;
+    padding: 5px 10px;
+    cursor: pointer;
+  }
+  
+  .cart-container button:hover {
+    background-color: #0056b3;
+  }
+  
+  
+  
+  
```

Comment:
@AYOPELUMI 
@Divinechi 
@adex-badr18 

Weldone, implementing the project requirements. You all agreed to use tailwind CSS for the implementation of the styles but here you went ahead to use vanilla CSS. Please kindly redo this part of the project.